app/assets/javascripts/application.js.erb

```diff
@@ -484,4 +484,23 @@ $(document).ready(function() {
     $("#story_guidelines").toggle();
     return false;
   });
+
+  $(document).on("focus", "textarea#comment", function() {
+    var trigger = false;
+
+    $(".comment_form_container").on("keydown", function(e) {
```

Comment:
I think this installs a duplicate event handler every time the field is focused, which would cause this to run more than once if the user types text, unfocuses the field, then focuses it again.